tools/roslaunch/scripts/roscore

```diff
@@ -49,6 +49,12 @@ def _get_optparse():
     parser.add_option("-v", action="store_true",
                       dest="verbose", default=False,
                       help="verbose printing")
+    parser.add_option("-w", "--numworkers",
+                      dest="nwork", default=3, type="int",
```

Comment:
The default should come from the constant in the code instead of being hard coded here.